include/lsst/meas/modelfit/CModel.h

```diff
@@ -442,6 +449,8 @@ struct CModelResult {
     PTR(afw::detection::Footprint) initialFitRegion;  ///< Pixels used in the initial fit.
     PTR(afw::detection::Footprint) finalFitRegion;    ///< Pixels used in the exp, dev, and linear fits.
 
+    LocalUnitTransform fitSysToMeasSys; ///< Transforms to the coordinate system where parameters are defined
```

Comment:
Does fitSysToMeasSys actually provide any functionality, i.e. does it actually 'fit' anything? It seems to be a class which only provides the data necessary for other functions to actually do the transforming. Consider renaming to avoid confusion, something such as SysToMeasSys(Info/Data/Mapping/etc)